docs/src/main/asciidoc/spring-cloud-commons.adoc

```diff
@@ -448,6 +448,9 @@ You can set:
 - `spring.cloud.loadbalancer.retry.maxRetriesOnSameServiceInstance` - indicates how many times a request should be retried on the same `ServiceInstance` (counted separately for every selected instance)
 - `spring.cloud.loadbalancer.retry.maxRetriesOnNextServiceInstance` - indicates how many times a request should be retried a newly selected `ServiceInstance`
 - `spring.cloud.loadbalancer.retry.retryableStatusCodes` - the status codes on which to always retry a failed request.
+
+NOTE:: For load-balanced retries, by default, we instantiate `AvoidPreviousInstanceRoundRobinLoadBalancer` that will select a different instance from the one previously chosen if available. You can switch back to using the basic `RoundRobinLoadBalancer` by setting the value of `spring.cloud.loadbalancer.retry.avoidPreviousInstance.enabled` to `false`.
```

Comment:
Change to "NOTE: For load-balanced retries, by default, we instantiate `AvoidPreviousInstanceRoundRobinLoadBalancer` to select a different instance from the one previously chosen, if available. You can switch back to using the basic `RoundRobinLoadBalancer` by setting the value of `spring.cloud.loadbalancer.retry.avoidPreviousInstance.enabled` to `false`."